src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -79,7 +79,7 @@ public ITest Build(Assembly assembly, IDictionary<string, object> options)
 #else
             log.Debug("Loading {0} in AppDomain {1}", assembly.FullName, AppDomain.CurrentDomain.FriendlyName);
 #endif
-            
+
```

Comment:
@CharliePoole I am not seeing these whitespace changes as they happen. Do you care about these? I can work on it.